app/code/Transoft/Blog/view/frontend/layout/blog_blog_index.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
Why layout( and action accordingly) is called blog **index**. Please rename to **view**

Reply:
renamed